src/eth1spec/trie.py

```diff
@@ -220,6 +224,9 @@ def c(J: Mapping[Bytes, Union[Bytes, Account, Receipt]], i: Uint) -> Bytes:
     # if leaf node
     if len(J) == 1:
         leaf = J[I_0]
+
+        I_1: Union[bytes, Uint]
```

Comment:
This seems a bit unfortunate and is a byproduct of having the storage trie be a `Dict[Bytes32, U256]`. I think when I rework this code I can improve some of these weird typings.